src/main/resources/templates/experiments.html

```diff
@@ -37,10 +38,50 @@ <h2 class="team-heading team-heading-current-team" th:text="#{exp.header}">Exper
+                                                        <h4 class="modal-title" id="expId"><span th:text="#{experiment.table.header.details}">Details</span></h4>
+                                                    </div>
+                                                    <div class="modal-body team-owner-info-body">
+                                                        <h4>Instructions to Access Nodes</h4>
+                                                        <ul>
+                                                            <ol>SSH into users</ol>
+                                                            <ol>SSH into the node by using its Qualified Name, e.g. <em>n0.demoexp.project.ncl.sg</em></ol>
+                                                        </ul>
+                                                        <h4>Info</h4>
+                                                        <form action="#">
+                                                            <textarea th:text="${realization.details}" class="realization" rows="30" cols="100"></textarea>
+                                                        </form>
+                                                    </div>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </td>
+                                </th:block>
                                 <td>
-                                    <a href="#" th:href="@{/start_experiment/{teamName}/{expId}(teamName=${expEntry.teamName}, expId=${expEntry.id})}" class="btn btn-standard" data-toggle="tooltip" data-placement="bottom" title="Start this experiment"><i class="fa fa-play btn-fa-green"></i></a>
-                                    <a href="#" th:href="@{/remove_experiment/{expId}(expId=${expEntry.id})}" class="btn btn-standard" data-toggle="tooltip" data-placement="bottom" title="Are you sure you want to remove the experiment from this team?"><i class="fa fa-trash btn-fa-red"></i></a>
-                                    <a href="#" th:href="@{/stop_experiment/{teamName}/{expId}(teamName=${expEntry.teamName}, expId=${expEntry.id})}" class="btn btn-standard" data-toggle="tooltip" data-placement="bottom" title="Stop this experiment"><i class="fa fa-stop btn-fa-green"></i></a>
+                                    <th:block th:each="realization : ${realizationMap.get(expEntry.id)}">
+                                        <span th:if="${realization.state == 'ACTIVATING'}">
+                                            <a href="#" data-toggle="tooltip" data-placement="bottom" title="resources allocation in progress..."><i class="fa fa-circle-o-notch fa-spin fa-fw btn-fa-loading" aria-hidden="true"></i></a>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)